bin/stop-dbs.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
should travis call this file after test?

Reply:
Travis recycles the containers. We don't have to worry about the `stop`. BTW, our Travis Mac build uses `brew`.

Reply:
No, Travis pretty much kills the instance that runs the tests once the tests are done ... and we aren't starting a docker containre on Travis but a service.